source/CommonJobs/CommonJobs.ContentExtraction.IFilterExtraction.Test/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net45" />
```

Comment:
Is this needed here? Is it possible that this dependency has been added to more projects than actually need it?